frontend/src/models/Robot.ts

```diff
@@ -8,6 +8,7 @@ export enum RobotStatus {
     Available = 'Available',
     Offline = 'Offline',
     Busy = 'Busy',
+    SafeZone = 'Safe Zone',
```

Comment:
Needs translation

Reply:
I think we use "trygg sone" as the translation btw @mrica-equinor, we're just missing this capitalisation